EIPS/eip-6454.md

```diff
@@ -4,8 +4,7 @@ title: Minimal Transferable NFT detection interface
 description: A minimal extension to identify the transferability of Non-Fungible Tokens.
 author: Bruno Škvorc (@Swader), Francesco Sullo (@sullof), Steven Pineda (@steven2308), Stevan Bogosavljevic (@stevyhacker), Jan Turk (@ThunderDeliverer)
 discussions-to: https://ethereum-magicians.org/t/minimalistic-transferable-interface/12517
-status: Last Call
-last-call-deadline: 2023-04-07
+status: Final
```

Comment:
The last commit changing content was May 13th. You'll need to restart the last call period.

```suggestion
status: Last Call
last-call-deadline: 2023-05-30
```

Reply:
I've set it to the 27th.